Modules/TRD/TRDQC.json

```diff
@@ -58,13 +58,10 @@
         "cycleDurationSeconds": "60",
         "maxNumberCycles": "-1",
         "dataSource": {
-          "type": "direct",
-          "query": "digits:TRD/DIGITS;tracklets:TRD/TRACKLETS;triggers:TRD/TRKTRGRD"
+          "type": "direct", 
```

Comment:
aren't the json files sensitive about spaces at end of lines?

Reply:
not sure, it worked when I went over 2 runs from catalog, 522392 and 529462. Still need to verify mc, and change dpg. dpg change will have to wait till new qc.